azure-pipelines.yml

```diff
@@ -106,8 +112,25 @@ jobs:
       choco install -y mingw --force --version=6.4.0
     displayName: 'Install 64-bit mingw for 64-bit builds'
     condition: eq(variables['BITS'], 64)
-  - script: python -m pip install numpy cython==0.29.13 pybind11 pytest pytest-timeout pytest-xdist pytest-env pytest-cov Pillow mpmath matplotlib
+  - script: python -m pip install numpy cython==0.29.13 pybind11 pytest pytest-timeout pytest-xdist pytest-env pytest-cov Pillow mpmath
     displayName: 'Install dependencies'
+  - powershell: |
+      If ($(PYTHON_VERSION) -ne '3.5') {
+          python -m pip install matplotlib==3.2.0rc1
+      }
+      else {
+          python -m pip install matplotlib
+      }
+    displayName: 'Install matplotlib'
+  - powershell: |
```

Comment:
Can you add a comment line above this saying why we're doing it? My understanding is that so we test the MacPython scipy wheels aren't failing tests. But just looking at the diff, and eventually later just looking at this file in an editor, the first impression might be that our `_distributor_init.py` file is broken and needs to be fixed/patched.

Reply:
Ok, I've added a few lines---the DLL resolution stuff was a complex discussion when it came up in NumPy, but hopefully I've provided the reader with some sense of motivation for the test.